pkg/cluster/k8sres.go

```diff
@@ -221,7 +222,22 @@ PatroniInitDBParams:
 	config.PgLocalConfiguration = make(map[string]interface{})
 	config.PgLocalConfiguration[patroniPGBinariesParameterName] = fmt.Sprintf(pgBinariesLocationTemplate, pg.PgVersion)
 	if len(pg.Parameters) > 0 {
-		config.PgLocalConfiguration[patroniPGParametersParameterName] = pg.Parameters
+		localParameters := make(map[string]string)
+		bootstrapParameters := make(map[string]string)
+		for param, val := range pg.Parameters {
+			if !isBootstrapOnlyParameter(param) {
```

Comment:
Can you please swap "if - else" branches ? Just makes it a little easier to read.

Reply:
makes sense